cmd/crank/render/cmd.go

```diff
@@ -41,6 +43,14 @@ func (c *Cmd) Run() error { //nolint:gocyclo // Only a touch over.
 		return errors.Wrapf(err, "cannot load Composition from %q", c.Composition)
 	}
 
+	warns, errs := comp.Validate()
+	for _, warn := range warns {
+		fmt.Fprintf(k.Stderr, "WARN(composition): %s\n", warn)
```

Comment:
Should we use `k.Errorf` here instead? I think it would be equivalent.

https://pkg.go.dev/github.com/alecthomas/kong#Kong.Errorf

Reply:
It will prefix the application name, I think we should review outputs across all commands to make it a more pleasant and cohesive experience